arcgis-ios-sdk-samples/Maps/Create and save a map/CreateSaveMapViewController.swift

```diff
@@ -145,6 +150,8 @@ class CreateSaveMapViewController: UIViewController, CreateOptionsViewController
                     }
                 }
             }
+            
+            AGSArcGISRuntimeEnvironment.apiKey = apiKey
```

Comment:
It seems to me like this should go inside the completion handler. The completion handler should always get called as in the event of connection issues, the request should time out.